libcudacxx/include/cuda/std/__type_traits/type_list.h

```diff
@@ -394,6 +456,52 @@ using __type_fold_right = __type_call<_List, detail::__type_fold_right_fn<_Init,
 template <class _List, class _Init, class _Fn>
 using __type_fold_left = __type_call<_List, detail::__type_fold_left_fn<_Init, _Fn>>;
 
+namespace detail
+{
+template <class _Ty>
+struct __type_remove_fn
+{
+  template <class _Uy>
+  using __apply = _CUDA_VSTD::_If<_CCCL_TRAIT(_CUDA_VSTD::is_same, _Ty, _Uy), __type_list<>, __type_list<_Uy>>;
```

Comment:
This is within namespace cuda::std, so I believe we can ommit the qualification
```suggestion
  using __apply = _If<_CCCL_TRAIT(is_same, _Ty, _Uy), __type_list<>, __type_list<_Uy>>;
```